### DISCUSSION TITLE
S1255-PC support

### DISCUSSION THREAD

**Discussion by wesleymostien** [yozik04/nibe#112] [General]
Hi, 

Just wondering if the model S1255-PC is supported, because I don't see it listed specifically, while other similar models are...

Thanks for the clarification!

**Comment by wesleymostien** [comment]
I didn't see it listed on the info pages (like https://www.home-assistant.io/integrations/nibe_heatpump)

But I see it in the code though : https://github.com/yozik04/nibe/blob/master/nibe/heatpump.py

**Comment by elupus** [comment]
The supported one is the non PC version. But i think the PC version  aught to work with that data set.

**Reply by wesleymostien** [reply]
I suppose it would, but are there extra values available regarding cooling for this one then?

**Reply by elupus** [reply]
Maybe. You would have to export the registry from it and add it here to see.

**Reply by wesleymostien** [reply]
I don't have the pump yet, it will be around May next year. If I remember to do so, I will add it here.

Any guidance or step-by-step instructions how to do it? Thanks!

**Reply by elupus** [reply]
You set the pump as english, go into the menus to find modbus settings, there you can export the database to a usb stick.

**Reply by wesleymostien** [reply]
Thanks, I will have a look once I have the pump.
